database/src/main/resources/db/migration/V5__create_restaurants_scheme.sql

```diff
@@ -0,0 +1,54 @@
+CREATE SCHEMA IF NOT EXISTS food;
+set search_path to food;
+
+CREATE TABLE IF NOT EXISTS cuisine(
+    cuisine_id SERIAL PRIMARY KEY,
+    cuisine_name VARCHAR(127) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS country(
+    country_id SERIAL PRIMARY KEY,
+    country_name VARCHAR(127) NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS city(
+    city_id SERIAL PRIMARY KEY,
+    city_name VARCHAR(127) NOT NULL,
+    country_id SERIAL not null references country(country_id) ON UPDATE CASCADE ON DELETE CASCADE
+);
+
+CREATE TABLE IF NOT EXISTS restaurant(
+    restaurant_id SERIAL PRIMARY KEY,
+    restaurant_name VARCHAR(127) NOT NULL,
+    restaurant_address VARCHAR(127) NOT NULL,
+    city_id SERIAL not null references city(city_id) ON UPDATE CASCADE ON DELETE CASCADE,
```

Comment:
Tego average'a zwraca API czy to nasz pomysł na feature?

Reply:
Tak, udostępnia nam to API